aten/src/ATen/native/native_functions.yaml

```diff
@@ -3755,7 +3755,7 @@
 - func: index_add_(Tensor(a!) self, int dim, Tensor index, Tensor source) -> Tensor(a!)
   variants: method
   dispatch:
-    CPU: legacy::cpu::_th_index_add_
+    CPU: index_add__cpu
```

Comment:
nit: I think it's clearer and more consistent to name this "index_add_cpu_", see e.g. "bernoulli_scalar_cpu_".  It's difficult to discern "__" and more consistent to always just end in an underscore.

Reply:
__cpu is also used in a lot of placed. But I agree the .._cpu_ is better. Maybe worth codemodding the others?